internal/controller/member/member.go

```diff
@@ -33,12 +32,12 @@ import (
 var (
 	ErrStartEtcd            = errors.New("start etcd failed")
 	ErrStartEtcdCanceled    = errors.New("etcd start canceled")
-	defaultEtcdStartTimeout = 2 * time.Minute
+	defaultEtcdStartTimeout = time.Minute
```

Comment:
one minute is too short

Reply:
I think leader election should be finished in 3 seconds, this timeout is to tolerate etcd downgrade